ui/templates/welcome.html

```diff
@@ -6,9 +6,26 @@
 
 <p>
```

Comment:
Okay, that makes sense. So despite the "empty" listing page, it wouldn't be a dead-end, because the next logical step (an import) is available right there.

In fact, all the headers should just be hidden if there are no imported courses, so they get nothing except the import link. I'll do that.